test/no-duplicate-modifiers.js

```diff
@@ -0,0 +1,58 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/no-duplicate-modifiers';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const ruleError = {ruleId: 'no-duplicate-modifiers'};
+const header = `const test = require('ava');\n`;
```

Comment:
Huh weird. Definitely a GitHub thing, I see it in my editor (and pretty sure no tests would have passed otherwise)